docs/api/samples/concept-scheme.json

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "dc": "http://purl.org/dc/elements/1.1/",
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#"
+    },
+    "@id": "http://example.com/taxonomy",
+    "dcterms:title": "Taxonomy Title",
+    "dcterms:description": "This is the description of my taxonomy!",
+    "dcterms:created": "2019-08-01",
+    "dcterms:updated": "2019-08-01"
```

Comment:
Will a concept scheme contain any of the top level concepts inside the `GET` request?

Reply:
Just in the same way that a `Concept` contains its `narrower`/`broader` properties and we give you the minimal required presentation information:

```
{
    "@type": "skos:ConceptScheme",
    "skos:hasTopConcept": [
         { "@id": "xxx", "skos:prefLabel": { ... } } 
     ]
}
```